### PR TITLE
Update pin names

### LINKED ISSUE #2133: Update pin names
Change all instances of

ID1/2/3/4 -> LA1/2/3/4
SEN -> RES
AN8 -> VOL
CNTR -> FRQ
W1/2 -> SI1/2

### PR DESCRIPTION
**Pull request by CodeWithSouma**
# Update pin names, Fix #2133 
Changed all instances of

ID1/2/3/4 -> LA1/2/3/4
SEN -> RES
AN8 -> VOL
CNTR -> FRQ
W1/2 -> SI1/2

### CODE REVIEW

**Comment by bessman** [comment]
You have a lot of false matches for the SEN -> RES pin change. Revert all changes related to SENSOR and SEND.

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
Did you just find and replaced the letters?

File: app/src/main/java/io/pslab/fragment/CompassDataFragment.java [hunk 1]
```diff
@@ -626,7 +626,7 @@ private void resetInstrumentData() {
 
     private void initiateCompassSensor(int type) {
 
-        CompassDataFragment.COMPASS_SENSOR s = CompassDataFragment.COMPASS_SENSOR.values()[type];
```
Comment (bessman) [inline app/src/main/java/io/pslab/fragment/CompassDataFragment.java:629]: Revert this.

File: app/src/main/java/io/pslab/fragment/BaroMeterDataFragment.java [hunk 2]
```diff
@@ -604,7 +604,7 @@ private void resetInstrumentData() {
     }
 
     private void initiateBaroSensor(int type) {
-        BaroMeterDataFragment.BARO_SENSOR s = BaroMeterDataFragment.BARO_SENSOR.values()[type];
```
Comment (bessman) [inline app/src/main/java/io/pslab/fragment/BaroMeterDataFragment.java:607]: Revert this.

**Review by bessman — CHANGES_REQUESTED** [review]
Revert all binary files.

File: app/src/main/java/io/pslab/fragment/BaroMeterDataFragment.java (hunk 2 above)
Comment (CodeWithSouma) [reply app/src/main/java/io/pslab/fragment/BaroMeterDataFragment.java:607]: @bessman @CloudyPadmal 
Sir please help me how t fix this problem

Comment (bessman) [reply app/src/main/java/io/pslab/fragment/BaroMeterDataFragment.java:607]: Change the line back to what it was before.

Comment (CodeWithSouma) [reply app/src/main/java/io/pslab/fragment/BaroMeterDataFragment.java]: > Change the line back to what it was before.

Sir i change these two line. But it is not work.How to fix it please help me.

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
1. Revert the `*.png` images back
2. Squash the commits into one (https://github.com/fossasia/pslab-android#how-to-git-squash)

**Comment by CodeWithSouma** [comment]
> 1. Revert the `*.png` images back
> 2. Squash the commits into one (https://github.com/fossasia/pslab-android#how-to-git-squash)

sir please check it. I refactor the code as you want.

**Review by bessman — CHANGES_REQUESTED** [review]
When reverting the PNGs you also reverted W1->SI1 and W2->SI2. Please reapply that part.

You appear to have applied your squash on top of the existing commits instead of replacing them. After a successful squash, the pull request should contain only one commit. Please try again.

**Comment by CodeWithSouma** [comment]
> When reverting the PNGs you also reverted W1->SI1 and W2->SI2. Please reapply that part.
> 
> You appear to have applied your squash on top of the existing commits instead of replacing them. After a successful squash, the pull request should contain only one commit. Please try again.

can you give me some easy trick to refactor w1 to s1 and w2 to s2.

**Comment by bessman** [comment]
> can you give me some easy trick to refactor w1 to s1 and w2 to s2.

grep for W1/2 and change to SI1/2 where applicable.

**Comment by CloudyPadmal** [comment]
Hello @CodeWithSouma , any update on this PR?

**Comment by CodeWithSouma** [comment]
@bessman @CloudyPadmal 
Sir please  checkout my PR. I also applied squash it works locally but when i push the code in remote it not work please help me how to squash all commit into one.

**Review by bessman — APPROVED** [review]
Looks good to me. Don't worry about squashing, we can do that when we merge.

**Review by CloudyPadmal — APPROVED** [review]

**Comment by CloudyPadmal** [comment]
@CodeWithSouma once you squashed, the git history changes in local. So you have to make a force push.

**Pull request merged**